x-pack/plugins/features/server/routes/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter } from '../../../../../src/core/server';
+import { LegacyAPI } from '../plugin';
+import { FeatureRegistry } from '../feature_registry';
+
+/**
+ * Describes parameters used to define HTTP routes.
+ */
+export interface RouteDefinitionParams {
+  router: IRouter;
+  featureRegistry: FeatureRegistry;
+  getLegacyAPI: () => LegacyAPI;
+}
+
+export function defineRoutes({ router, featureRegistry, getLegacyAPI }: RouteDefinitionParams) {
+  router.get(
+    { path: '/api', options: { tags: ['access:features'] }, validate: false },
```

Comment:
__note:__ hmm, looking at the failing tests. It seems `tags` in NP doesn't work as we expect. Investigating....